codequest_pipes.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |spec|
   spec.name    = 'codequest_pipes'
-  spec.version = '0.3.1.1'
+  spec.version = '0.3.2'
```

Comment:
what about `0.3.2.0` to keep consistent?

Reply:
We want to use [semantic versioning](https://semver.org/) - that's why it consists of only three numbers now.